src/odemis/util/transform.py

```diff
@@ -224,10 +243,15 @@ def to_physical_space(
 
 
 def to_pixel_index(
```

Comment:
I previously liked that these two functions `to_physical_space` and `to_pixel_index` were really clear and simple; the input and output are both a list of coordinates. 

We could also consider having a to_physical_space and a to_pixel_index method on the `GeometricTransform` class.

Reply:
Then I'd rather make it separate functions. Or just not merge this PR for now and see if it is really that necessary

Reply:
I agree with Thera that the original simple version of the function is nicer to read. Object-oriented pattern would indeed suggest to put such function as a method on the class.
That said, it might not be super easy due to (cycle-)dependencies. I'd say it's also fine to create new functions named something like `tform_to_physical_space` and `tform_to_pixel_index`, for just this purpose.

Reply:
Érics idea of having two functions like `tform_to_physical_space` and `tform_to_pixel_index`, sounds like a good idea to me.